axelrod/tests/unit/test_adaptive.py

```diff
@@ -0,0 +1,50 @@
+"""Test for the Adaptive strategy."""
+
+import axelrod
+
+from .test_player import TestHeadsUp, TestPlayer
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
+
+class TestAdaptive(TestPlayer):
+
+    name = "Adaptive"
+    player = axelrod.Adaptive
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Test initial play sequence
+        self.responses_test([], [], [C] * 6 + [D] * 5)
```

Comment:
Could we have some specific tests for some of the methods: for example check that if the strategy does not know the game that the default one is used?